src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -92,6 +92,22 @@ def format_age(data: pd.DataFrame) -> pd.Series:
     return data["age"].astype(int)
 
 
+def format_copy_age(data: pd.DataFrame) -> pd.Series:
+    no_copy_idx = data.index[data["copy_age"].isna()]
+    column = data["copy_age"].astype(str)
+    column = column.apply(lambda row: row.split(".")[0])
+    column.loc[no_copy_idx] = np.nan
```

Comment:
Isn't this already NaN by definition of `no_copy_idx`?

Reply:
See other comment.